Mathlib/Algebra/Polynomial/Roots.lean

```diff
@@ -0,0 +1,829 @@
+/-
+Copyright (c) 2018 Chris Hughes. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Chris Hughes, Johannes Hölzl, Scott Morrison, Jens Wagemaker, Johan Commelin
+-/
+
+import Mathlib.Algebra.Polynomial.RingDivision
+import Mathlib.RingTheory.Localization.FractionRing
+
+#align_import data.polynomial.ring_division from "leanprover-community/mathlib"@"8efcf8022aac8e01df8d302dcebdbc25d6a886c8"
```

Comment:
Should we have this line twice? Or should it only be in this file, because importing this file is the moral equivalent of importing `ring_division` in Lean3.

Reply:
We should have the line twice, you are right. Thanks!

Reply:
Indeed we already have it twice!